src/device_trezor/device_trezor_base.hpp

```diff
@@ -229,6 +229,12 @@ namespace trezor {
       return m_features;
     }
 
+    uint64_t get_version() {
+      CHECK_AND_ASSERT_THROW_MES(m_features, "Features not loaded");
+      CHECK_AND_ASSERT_THROW_MES(m_features->has_major_version() && m_features->has_minor_version() && m_features->has_patch_version(), "Invalid Trezor firmware version information");
+      return pack_version(m_features->major_version(), m_features->minor_version(), m_features->patch_version());
+    }
```

Comment:
const ?

Reply:
it is a plain function in the namespace. so OK as it is?